Sources/TootSDK/TootClient/TootClient+Post.swift

```diff
@@ -242,6 +242,20 @@ extension TootClient {
 
 }
 
+extension TootClient {
+
+    /// Translate the post content into some language.
+    public func getPostTranslation(id: String, params: PostTranslationParams? = nil) async throws -> Translation {
```

Comment:
```suggestion
    public fund publishPostTranslation(id: String, params: PostTranslationParams? = nil) async throws -> Translation {
```

Reply:
I can commit this but I believe it's wrong, you're not publishing a translation you're querying a translation.

Reply:
Oh you're right of course!